tests/client/layout_position_tests.cpp

```diff
@@ -58,4 +60,47 @@ TEST_CASE("Layout Position Tests", "[layout-position]")
     
     REQUIRE(true); // Placeholder - actual testing requires DOM setup
   }
+
+  SECTION("Static position should ignore top/left/right/bottom properties")
+  {
+    // This test verifies that when position: static is used,
+    // the top, left, right, and bottom properties are ignored
+    // as per CSS specification
+    
+    using namespace client_cssom::values;
+    
+    // Create a computed style with position: static and inset values
+    ComputedStyle staticStyle;
+    computed::Context context;
```

Comment:
@EndlessJour9527 See https://github.com/M-CreativeLab/jsar-runtime/blob/main/tests/client/css_transform_parser_tests.cpp#L500, writing it this way directly will cause `TransmuteUnitTest` to crash.